server/api/call.go

```diff
@@ -18,6 +18,8 @@ const (
 	// CallTypeCancel is used for for the (rare?) case of when the form with
 	// SubmitOnCancel set is dismissed by the user.
 	CallTypeCancel = CallType("cancel")
+	// CallTypeLookup is used to fetch items for dynamic select elements
+	CallTypeLookup = CallType("lookup")
```

Comment:
Let's file a separate ticket if we haven't yet to review and finalize the lookup protocol. I am not happy with where we are ATM, but it works for this PR.